test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -250,6 +260,28 @@ Describe "TabCompletion" -Tags CI {
             Pop-Location
         }
 
+        It "TabCompletion should be case-insensitive for file names on Windows and MacOS" -Skip:($IsLinux) {
+            Push-Location -Path $tempDir
+            $res = TabExpansion2 -inputScript $oneSubDirLowerTest -cursorColumn $oneSubDirLowerTest.Length
+            $res.CompletionMatches | Should -HaveCount 1
+            $res.CompletionMatches[0].CompletionText | Should -BeExactly ".${separator}$oneSubDirTest"
+
+            $res = TabExpansion2 -inputScript $oneSubDirUpperTest -cursorColumn $oneSubDirUpperTest.Length
+            $res.CompletionMatches | Should -HaveCount 1
+            $res.CompletionMatches[0].CompletionText | Should -BeExactly ".${separator}$oneSubDirTest"
+        }
+
+        It "TabCompletion should be case-sensitive for file names on Unix" -Skip:(!$IsLinux) {
+            Push-Location -Path $tempDir
+            $res = TabExpansion2 -inputScript $oneSubDirLowerTest -cursorColumn $oneSubDirLowerTest.Length
+            $res.CompletionMatches | Should -HaveCount 2
```

Comment:
This should still be `-HaveCount 1`?  You're testing that "onesubd" and "oneSubD" both return "onesubdir", right?

It seems we should have a test for Linux for a directory structure like:

```none
testfolder
testFolder
```

and tabcomplete off `test` should return two results

Reply:
This additional test case is still needed.

Reply:
@SteveL-MSFT It is already there - see line  226. So I removed last commit. Seems my solution is not full - the count should be 2 on . 
How can I get the artifact (compiled PowerShell) from CI Travis?

Reply:
@iSazonov you're right, I misunderstood that test.  Did a little investigation and it appears that the change to GetFileSystemEntries() is not sufficient as a call to Resolve-Path earlier only returns case-sensitive matches on Linux.

Reply:
Should we open new issue to discuss making `Resolve-Path` case-insensitive? This is most likely to affect globbing whole.

Reply:
@iSazonov we should probably have a new issue for `Resolve-Path`.  May need to add a `-CaseInsensitive` switch to maintain backwards compatibility.